src/Main.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * @(#)<main.Java.java        <project-version> <2024/10/30>
+ *
+ * <Task management application>
+ *
+ * Built using <IntelliJ> IDE Version: <2023.2.5>
+ * OpenJDK Runtime Environment <OpenJDK>-<17.0.9>
+ *
+ */
```

Comment:
I believe there's been an misunderstanding of the comments specified in the CONTRIBUTING.md. But you did a great job in write as recommended way. Here's are the few suggested changes that you need to make to make this alright.

> Tłumaczenie
> ---
>
> Uważam, że doszło do nieporozumienia w komentarzach podanych w pliku CONTRIBUTING.md. Ale wykonałeś świetną robotę, pisząc zgodnie z zaleceniami. Oto kilka sugerowanych zmian, które musisz wprowadzić, aby wszystko było w porządku.
>
>> Przetłumaczone przez Tłumacz Google

```suggestion
/*
 * @(#)Main.java       1.0.0 2024/10/30
 *
 * Task management application
 *
 * Built using IntelliJ IDE Version: 2023.2.5
 * OpenJDK Runtime Environment OpenJDK-17.0.9
 *
 */
```